docs/cli-commands.md

```diff
@@ -538,7 +538,7 @@ The following options are available:
 - `--max_file_size=int` (optional, default: `2000`): max. multimedia file size allowed, in MB (max: 100'000)
 - `--latest` (optional): 
   instead of the latest deployed version,
-  use the latest development version of DSP-API (from the `main` branch)
+  use the latest development version of DSP-API/DSP-APP (from the `main` branch)
```

Comment:
This is a user-friendly abstraction of what is really happening. Already on line 526, we don't tell the entire truth.